hackerrank/simple-array-sum/challenge.js

```diff
@@ -0,0 +1,12 @@
+// Solve the challenge in this JavaScript file
+
+const simpleArraySum = (ar) => {
+  // Write your code here
+  let sum = 0;
+  for (let i = 0; i < ar.length; i += 1) {
+    sum += ar[i];
+  }
```

Comment:
Let's use `forEach` for this, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach